File: Competition/BOJ_Bundle_in_Math/Vol_1/H.cpp
// problem: H번 - A+B - 10 (제1편)
// id:
// time taken:
#include <bits/stdc++.h>
using namespace std;
int main(void)
{
    int r, a, b;
    bool ca = false, cb = false;
    for (int i = 1; i < 9; i++)
    {
        cout << "? A " << i << endl;
        cin >> r;
        if (r)
        {
            a = i;
            ca = true;
            break;
        }
    }
    if (!ca)
        a = 9;
    for (int i = 1; i < 9; i++)
    {
        cout << "? B " << i << endl;
        cin >> r;
        if (r)
        {
            b = i;
            cb = true;
            break;
        }
    }
    if (!cb)
        b = 9;
    cout << "! " << a + b;

    return 0;
}

File: Competition/Solvedac_Arena/solved.ac Grand Arena #1/H.py
# problem: H번 - 행렬 연산 (행렬 계산하기)
# id:
# time taken:

n, m, q = map(int, input().split())
mt = [[0 for j in range(m)] for i in range(n)]

for _ in range(q):
    o, r, v = map(int, input().split())
    if o == 1:
        for i in range(m):
            mt[r - 1][i] += v
    else:
        for i in range(n):
            mt[i][r - 1] += v

for i in range(n):
    ls = list(map(str, mt[i]))
    print(" ".join(ls))
